packages/dotcom-ui-header/src/header.scss

```diff
@@ -27,3 +27,40 @@
 .n-typeahead {
   display: none;
 }
+
+// The styles below are part of an AB test
```

Comment:
**Note:** These styles are not designed to be perfect for a future after the test. If the test is sucessfull then the styles should correctly designed within o-header.